### DISCUSSION TITLE
Show wallet height

### DISCUSSION THREAD

**Discussion by joelrb** [Chia-Network/chia-blockchain#7968] [Ideas]
The wallet has been syncing for like 3 days already. I don't know when it will stop.

It would be helpful to show the height of the wallet so I'd know until when wallet will sync.

![Screen Shot 2021-08-06 at 10 11 54 PM](https://user-images.githubusercontent.com/225284/128523528-a8eca041-666a-45a9-bc2f-4725fb2add3c.png)

Node is fully sync as I copied the db from another node.

![Screen Shot 2021-08-06 at 10 13 38 PM](https://user-images.githubusercontent.com/225284/128523599-99dd7d42-8389-46e3-9f8d-d0c281548def.png)

**Comment by keliew** [comment]
Delete the _wallet_ db, and let it re-sync again.

Don't delete the blockchain db.
